### DISCUSSION TITLE
Magisk mention

### DISCUSSION THREAD

**Discussion by Vistaus** [community/community#43126] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Your behavior on the Magisk repo aside (https://github.com/topjohnwu/Magisk/issues/6501#issuecomment-1368957039), why did you mention me as part of the Magisk team? I'm barely a user, let alone part of the Magisk team. So I'm really unsure as to why you think I am.

**Comment by jfullstackdev** [comment]
are you referring to this organization ? 

https://github.com/Magisk-Modules-Repo

you can contact them there

**Reply by Vistaus** [reply]
No, my feedback was meant for @admin-elvistrujanovic, but I guess something went wrong when posting.

Edit: I see. I clicked on ‘Send Feedback’ on his profile, thinking it would be feedback for him, but I guess it's about feedback for the beta. Sorry for my mistake.

**Reply by joeyoropesa-dev** [reply]
What's happening? Can I know how can I fix my problem. Will I be banned from the whole GitHub because I have a very bad feeling

**Reply by Vistaus** [reply]
I'm referring to your comment on https://github.com/topjohnwu/Magisk/issues/6501#issuecomment-1368957039

**Reply by joeyoropesa-dev** [reply]
> I'm referring to your comment on https://github.com/topjohnwu/Magisk/issues/6501#issuecomment-1368957039

Yes, I saw that right now 😔 - I really made a big mistake, I'm so sorry to you and all people there I mentioned and for my rude behavior to the creator of that project and others via that issue I made before

**Comment by joeyoropesa-dev** [comment]
My error (mistake) and sorry for that. I was created a big mistake to mention anyone including the staff team for this. I'm in a big trouble now by Magisk team and of course, other members there 😔

And maybe I can lost my GitHub account too and I hope that the ban will not be perm.

**Reply by Vistaus** [reply]
But again: I'm NOT part of the Magisk staff, so I still don't understand why you included me and how you even found my user name. Did you just search for random people to include?

**Reply by joeyoropesa-dev** [reply]
> But again: I'm NOT part of the Magisk staff, so I still don't understand why you included me and how you even found my user name.

When I was tried to ping the whole staff team on Magisk repo, I just type @ and I was think before that it will show me only the staff members there, not the users too 😓 - but I know that Discord feature "@" is not limited to show only staff team, it show every person on the web in the created issue

**Comment by Vistaus** [comment, marked as answer]
Well, this seems to be solved now, so let's forget about it. Again, to the GitHub staff: I'm sorry for the disruption - I misinterpreted where ‘Send Feedback’ would take me. I know now, so this won't happen again.
